src/modm/ui/display/monochrome_graphic_display_horizontal_impl.hpp

```diff
@@ -18,22 +18,22 @@ template<int16_t Width, int16_t Height>
 void
 MonochromeGraphicDisplayHorizontal<Width, Height>::setPixel(int16_t x, int16_t y)
 {
-	if ((x < Width) and (y < Height)) { buffer[y][x / 8] |= (1 << (x % 8)); }
+	if ((x < Width) and (y < Height)) { this->buffer[y][x / 8] |= (1 << (x % 8)); }
```

Comment:
Seems that no test/example code uses this class. Adding `this->` fixes the build, but I didn't dig to why didn't it work in the first place.

Reply:
My failure on missing `this->` whilst refacture. You're the first applying the vertical monochrome since then.

Reply:
Any quick explanation why was it needed in the first place?

Reply:
I found some explanation myself:
- https://www.cs.technion.ac.il/users/yechiel/c++-faq/nondependent-name-lookup-members.html
- https://eli.thegreenplace.net/2012/02/06/dependent-name-lookup-for-c-templates

Interesting read!